docs/kratos/guides/deploy-kratos-example.mdx

```diff
@@ -6,55 +6,51 @@ title: Deploy to production
-- Installation and configuration of [Nginx](https://www.nginx.com/)
-- Installation and configuration of [Ory Kratos Node.js UI Reference](https://github.com/ory/kratos-selfservice-ui-node)
+This guide explains how to set up and run Ory Kratos in an exemplary production environment. It uses Postgres as database, Nginx
+as reverse proxy, Digital Ocean as cloud provider, and the
+[Ory Kratos Node.js UI Reference](https://github.com/ory/kratos-selfservice-ui-node) as user interface. You can use another
+[relational database](https://www.ory.sh/docs/ecosystem/deployment#data-storage-and-persistence), a different reverse proxy,
+deploy on any other cloud host, and [spin up a custom interface in your favorite language](https://www.ory.sh/docs/kratos/sdk/) -
+this is just an example!
 
 ## Create a Droplet
 
-Spin up a droplet with the following configuration:
+Spin up a Droplet (virtual machine) with the following configuration:
 
 - **OS**: Ubuntu 20.04
 - **Plan**: Basic
 - **CPU options**: Regular with SSD
 - **RAM**: 1Gb
 - **SSD**: 25Gb
 - **VPC network**: default
-- **Authentication**: SSH Keys. Don't forget to add your own SSH key
+- **Authentication**: SSH Keys. Don't forget to add your SSH key.
 - **Region**: Choose your region
 
 :::note
 
-This example shows a single configuration on a small droplet. The configuration of a virtual machine (VM) may vary depending on
-the scale of your application.
+This example shows a basic configuration of Ory Kratos on a single virtual machine (VM). The configuration of a VM may vary
+depending on the scale of your application.
 
 :::
 
-Wait for the virtual machine (VM) to start up and copy the IP address.
+Wait for the Droplet to start up and copy the IP address.
 
 This guide uses `accounts.example.com` as a hostname to run Ory Kratos. Replace it with `<something>.<your-domain>` for your
-purposes. You need to configure DNS, create an `A type` record, and point it to the VM's IP.
+purposes. At your hosting provider, configure DNS, create an A type record, and point it to the Droplet IP.
 
 ## Install required dependencies
 
-Connect to your [droplet via SSH](https://docs.digitalocean.com/products/droplets/how-to/connect-with-ssh/) or use the
+Connect to your [Droplet via SSH](https://docs.digitalocean.com/products/droplets/how-to/connect-with-ssh/) or use the
 [Droplet Console](https://docs.digitalocean.com/products/droplets/how-to/connect-with-console/).
 
-First upgrade all packages in your VM.
+First, upgrade all packages in your Droplet.
 
 ```bash
 apt-get update && apt-get upgrade -y
 ```
 
-The default version of [Node.js](https://nodejs.org/en/) is v10.19.0 on Ubuntu 20.04, we need to install a newer version. You can
-use the following script to install Node.js 16.x on a Ubuntu system.
+Since the default version of [Node.js](https://nodejs.org/en/) is outdated we need to install a newer version. You can use the
+following script to install Node.js 16 on a Ubuntu system.
```

Comment:
This sounds a bit weird. Maybe re-phrase it to say that the user needs a specific version of Node.js to do all the things we talk about in this guide? I mean, 16.14.2 is bound to be outdated very soon (as a matter of fact 16.16.0 seems to be the newest v.) Also, would be great if you could get rid of the "we" form ;)

Reply:
Same comment applies to the Hydra guide.

Reply:
You dont need a specific version to do the guide - you just need the 16.x version installed with the script below, it will install the latest version. 

We had the same discussion on the update to hydra guide 😉

Reply:
Oh, ok, totally forgot about it